Library/Homebrew/extend/os/mac/ENV/std.rb

```diff
@@ -0,0 +1,141 @@
+module Stdenv
+  # @private
+  def setup_build_environment(formula = nil)
+    setup_generic_build_environment formula
+
+    if MacOS.version >= :mountain_lion
+      # Mountain Lion's sed is stricter, and errors out when
+      # it encounters files with mixed character sets
+      delete("LC_ALL")
+      self["LC_CTYPE"]="C"
+    end
+
+    # Add lib and include etc. from the current macosxsdk to compiler flags:
+    macosxsdk MacOS.version
+
+    if MacOS::Xcode.without_clt?
+      append_path "PATH", "#{MacOS::Xcode.prefix}/usr/bin"
+      append_path "PATH", "#{MacOS::Xcode.toolchain_path}/usr/bin"
+    end
+
+    # Leopard's ld needs some convincing that it's building 64-bit
+    # See: https://github.com/mistydemeo/tigerbrew/issues/59
+    if MacOS.version == :leopard && MacOS.prefer_64_bit?
+      append "LDFLAGS", "-arch #{Hardware::CPU.arch_64_bit}"
+
+      # Many, many builds are broken thanks to Leopard's buggy ld.
+      # Our ld64 fixes many of those builds, though of course we can't
+      # depend on it already being installed to build itself.
+      ld64 if Formula["ld64"].installed?
+    end
+  end
+
+  def homebrew_extra_pkg_config_paths
+    ["#{HOMEBREW_ENV_PATH}/pkgconfig/#{MacOS.version}"]
+  end
+
+  # Sets architecture-specific flags for every environment variable
+  # given in the list `flags`.
+  # @private
+  def set_cpu_flags(flags, default = DEFAULT_FLAGS, map = Hardware::CPU.optimization_flags)
+    generic_set_cpu_flags(flags, default, map)
```

Comment:
Just a tiny style nit, but I think it would help readability if there was always an empty line after `generic_*(<arguments>)`, i.e. here and in all following methods that modify the generic behavior.